frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayProjectChanges.java

```diff
@@ -263,7 +267,7 @@ private void renderReleaseIdToUsage(StringBuilder display) {
                    additions.getReleaseIdToUsage().keySet(),
                    changedReleaseIds);
 
-           LinkedReleaseRenderer renderer = new LinkedReleaseRenderer(display, tableClasses, idPrefix, actual.getCreatedBy());
+           LinkedReleaseRenderer renderer = new LinkedReleaseRenderer(display, tableClasses, idPrefix, userEmail);
```

Comment:
This email address is used to load the user on behalf of whom the releases are requested in `LinkedReleaseRenderer`. Two problems with that:
- I don't think current moderator should load releases on behalf of someone else.
- The old code throws an error if the user named in `createdBy` does not exist in the system. Which can happen when you import data from a legacy application, for example. The consequence in that case, is that one cannot review and accept or decline the moderation request at all.